app/package.json

```diff
@@ -4,6 +4,8 @@
   "private": true,
   "dependencies": {
     "@arkhn/ui": "^1.10.2",
+    "@blueprintjs/core": "^3.40.0",
+    "@blueprintjs/icons": "^3.25.0",
```

Comment:
Yeah, I find the blueprint icons very useful in our usecase, when our apps speak about tables, columns, etc. Blueprint has a lot of cool icons around this topic. Is it "costly" to keep using these icons instead of material-ui's one ? 

If yes, I can also check material icons again, and do some ajustments in Figma in order to fit. 
But more generally I like the idea (maybe in the future) that even if we use a UI library (Material here), we can still have our own "touch" above (could be icons, colors, having the lib components a bit customized...).

What do you think ?

Reply:
not very costly, it's fine by me, just wanted to make sure :)

Reply:
Alright cool then !